airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -102,18 +105,7 @@ def ti_run(
         .with_for_update()
     )
     try:
-        (
-            previous_state,
-            dag_id,
-            run_id,
-            task_id,
-            map_index,
-            next_method,
-            try_number,
-            max_tries,
-            next_method,
-            next_kwargs,
-        ) = session.execute(old).one()
+        ti = session.execute(old).one()
```

Comment:
Rather than using the row as a tuple, make use of the name attribute access that SQLA already gives us. Just makes it a bit easier.